.travis.yml

```diff
@@ -31,6 +31,8 @@ matrix:
       env: TOXENV=py2-cover,py3-cover,coverage
     - python: "3.6"
       env: TOXENV=docs
+    - python: "3.6"
+      env: TOXENV=lint
```

Comment:
Add python 3.7. [See Pyramid's](https://github.com/Pylons/pyramid/blob/master/.travis.yml#L25-L34):

```
        - python: 3.7
          env: TOXENV=py37
          dist: xenial
          sudo: true
        - python: 3.8-dev
          env: TOXENV=py38
          dist: xenial
          sudo: true
    allow_failures:
        - env: TOXENV=py38
```

Reply:
This is only for the lint